src/main.js

```diff
@@ -146,6 +168,7 @@ function renderOverlay() {
     el.setAttribute('style', ['background-color:#222', 'color:#fff', 'height:100%', 'font-family:system-ui'].join(';'));
     el.innerHTML = `<header style='position:fixed;top:0;left:0;right:0;display:flex;align-items:center;justify-content:space-between;padding:1rem;height:2.5rem;background-color:#333;z-index:1;'>\
         <div style='font-family:monospace;font-size:1.5em;cursor:pointer;' onclick='location.reload()'>InstagramUnfollowers</div>\
+        <button class='copyListToClipboardButton' style='cursor:pointer;' onclick='copyListToClipboard()' disabled>Copy List to Clipboard</button>\
```

Comment:
I would give it a bit of styling.
The default button is rather ugly IMO.
How about something like `style='background:none;color:white;border: 1px solid white;border-radius:15px;padding:0.5em;cursor:pointer'` ?